### DISCUSSION TITLE
Path to snapshots with docker install using HACS integration

### DISCUSSION THREAD

**Discussion by mcondren** [blakeblackshear/frigate#2010] [Ask A Question]
Hello,

I have frigate running in a container with a volume mounted so it stores every thing in /mnt/media. Ie. /mnt/media/frigate/clips/

The HACS frigate integration seems to know this because in the HA media viewer/player I see all my recordings, clips, and snapshots. 

I assume the integration has told HA that there is a media_source at /mnt/media/. However, this media_source doesn’t get added to configuration.yml with the other media sources so I am not sure how to reference that path in a notification. 

I tried 
attachment: /mnt/media/frigate/clips/{{ trigger.payload_json['after']['camera']}}-{{ trigger.payload_json['after']['id'] }}.jpg   

but it can’t find that location. Is frigate creating a custom media_source for the media player that can be referenced for snapshots as well?

**Comment by mcondren** [comment]
If the integration is creating a classic media_source I would expect it to be something like this in configuration.yaml. 

```
media_dirs:
  frigate: /mnt/media/frigate
```

And then I would reference that media source as per the home assistant docs as /media_source/frigate/clips/xxx.jpg but I’m not sure if that is what the integration is doing.

**Comment by blakeblackshear** [comment]
The media source in the integration doesn't work like that at all. It uses the http api from frigate and works even when frigate runs on an entirely separate machine. You should be using the http endpoints to pull snapshots, etc from frigate in your integrations. Or you can simply use the notification urls that the integration provides without needing to do any kind of downloading or copying to your www folder.

**Comment by mcondren** [comment]
Thanks @blakeblackshear.  I had seen this in the new docs (`https://docs.frigate.video/configuration/snapshots`) and had hoped I could use the local location to get around the restriction notify services have in HA now, that don’t allow URL’s outside the network.

**Reply by blakeblackshear** [reply]
You can. You have to download from frigate's http api before using the notification service

**Reply by mcondren** [reply]
Thanks. I ended up adding the clips folder as the /media volume in my docker compose file. (As read-only of course) and then it let me use /media in notifications.
